### DISCUSSION TITLE
CAPTURE HANDSHAKE Too long, I've tried several times and for several hours and still the words wpa handshake above don't appear on the right.

### DISCUSSION THREAD

**Discussion by Joyfelixx** [aircrack-ng/aircrack-ng#2498] [Q&A]
I capture handshake Until Hours and still not finished ![IMG20230306032313.jpg](https://user-images.githubusercontent.com/126768821/223066560-fd9b4c01-f69e-4356-8400-faf9ddd2b471.jpg)

I tried several times and waited for several hours but the WPA handshake didn't appear on the right

**Comment by Mister-X-** [comment]
You've tried for one hour, not "Hours".

That indicates a technique issue. Check out the WPA-PSK tutorial in our website as well as the analysis of the WPA capture.

**Reply by Joyfelixx** [reply]
I have tried many times but the result is the same

**Comment by ChillVibesMushroom** [comment]
What if the ap isn’t showing connected devices but data is flowing and there is a connected device

**Reply by Mister-X-** [reply]
Then I guess your wireless adapter can't capture the data possibly because it's 802.11n and the traffic is 802.11ac.

A couple things:
- This doesn't need to be picture. It's all text, so you can copy/paste it here (and put it between 3 backticks
- You need to provide more information about your system and the wireless hardware involved

**Reply by ChillVibesMushroom** [reply]
Can I determine with wireshark what encryption of 802.11 I’m capturing does wireshark only capture a certain type

**Reply by Mister-X-** [reply]
Yes, you can determine what encryption is being used.

**Reply by ChillVibesMushroom** [reply]
What if I installed a tool called airpwn-ng and had to make some files for an adapter would that fuck up my adapter and make it to where only some networks showed devices

that doesn’t make sense does it

**Reply by Mister-X-** [reply]
Assuming  you're saying you did this already, a tool injecting/capturing packets won't screw up a device.

It makes perfect sense that if your adapter is 802.11n, it can't see 802.11ac/ax (traffic being 802.11ax). You still haven't said what chipset it is.

**Reply by ChillVibesMushroom** [reply]
RTL8821CE 802.11ac wireless supports both 5 and 2 GHz but aircrackng won’t show devices it will only show that data is being sent I need to see devices

**Reply by ChillVibesMushroom** [reply]
What parameters can I use to make sure that I’m forcing the traffic to be sniffed or something

**Reply by Mister-X-** [reply]
As I said, maybe the traffic is 802.11ax. Or using features that this adapter doesn't support. Or even multiple streams (this device is just one stream).

**Reply by ChillVibesMushroom** [reply]
I was able to capture traffic in the past on the same router is it possible someone configured the access point that caused what was mentioned above

**Reply by Mister-X-** [reply]
It's possible these devices were 802.11n only and using 1 stream.

**Reply by ChillVibesMushroom** [reply]
I can put my router in 802.11n mode Ill test it to see whats up but why is it that we shouldnt put our cards in monitor mode with airmon-ng???

**Comment by ZerBea** [comment]
Removed my comment, because I assume that you are not the admin of this router:
"... is it possible someone configured the access point that caused what was mentioned above"

**Reply by ChillVibesMushroom** [reply]
I’m also trying to capture traffic and view it unencrypted with aircrackng tbh

**Comment by ChillVibesMushroom** [comment]
I just reset my router lol but thanks anyways man
